cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ZookeeperLeaderFinder.java

```diff
@@ -59,13 +59,13 @@ public ZookeeperLeaderFinder(Vertx vertx, SecretOperator secretOperator, Supplie
         this.backOffSupplier = backOffSupplier;
     }
 
-    /*test*/ NetClientOptions clientOptions(Secret coCertKeySecret, Secret clusterCaCertificateSecret) {
+    /*test*/ NetClientOptions clientOptions(Reconciliation reconciliation, Secret coCertKeySecret, Secret clusterCaCertificateSecret) {
```

Comment:
I think you can pass the `reconciliation` here through the constructor to avoid doing it in every method?